Chummer/frmCharacterOptions.cs

```diff
@@ -740,6 +740,7 @@ private void PopulateCustomDataDirectoryTreeView()
                     if (objCustomDataDirectory.Item1 is CustomDataDirectoryInfo objInfo)
                     {
                         objNode.Text = objInfo.Name;
+                        objNode.ToolTipText = objInfo.GetManifestData(objInfo.Path);
```

Comment:
This particular arrangement works right now because description data is all we'd be using. However, the moment stuff like dependency data or author data or version data would also be used, these tooltips are probably going to get crowded and/or hard to follow with your eyes. Maybe it would be better to have this data appear in a dedicated section within this tab of `frmCharacterOptions` instead of as tooltips. If we do that, we could then also use the tooltips for other stuff, e.g. explanations as to why an enabled custom ruleset might be highlighted a certain color when it has an unmet dependency.

Reply:
I would keep this for now, because it is the simplest solution at the moment.
We have to rework this later though.

Something akin to how spell information are displayed could work well. So a layout panel of some sort, that shows the description of the selected `treNode`.